### PR TITLE
contrib/kvantum: new package (1.0.10)

### PR DESCRIPTION
**Pull request by nekopsykose**

### CODE REVIEW

File: contrib/kvantum/template.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+]
+makedepends = [
+    "kwindowsystem-devel",
+    "libx11-devel",
+    "libxext-devel",
+    "qt6-qtbase-devel",
+    "qt6-qtsvg-devel",
+    "qt6-qttools-devel",
+]
+depends = ["hicolor-icon-theme"]
```
[the remark this answers is not in the document]
Comment (nekopsykose) [reply contrib/kvantum/template.py:20]: qt6-qtbase is the base for any qt app, including headless ones that don't have guis. unless you mean -gui?

[the remark this answers is not in the document]
Comment (nekopsykose) [reply contrib/kvantum/template.py]: proposed pushed

[the remark this answers is not in the document]
Comment (q66) [reply contrib/kvantum/template.py]: -gui is probably better, however it doesn't matter much (hicolor-icon-theme is a single file with a few lines)

**Comment by nekopsykose** [comment]
shit passes just ci takes forever on qt

**Comment by nekopsykose** [comment]
added the qt6 symlinks here too

**Pull request merged**